client/me/purchases/index.js

```diff
@@ -13,25 +13,25 @@ import billingController from 'me/billing-history/controller';
 import meController from 'me/controller';
 import { siteSelection } from 'my-sites/controller';
 import controller from './controller';
-import paths from './paths';
+import * as paths from './paths';
```

Comment:
It seems we're using all imports here, so we won't save anything if we replace the `*` with particular imports. It might be useful though to list all specific imports here in case we introduce a new path that we don't want to unnecessarily import.

Reply:
see above response :smile: